hedera-node/src/main/java/com/hedera/services/context/properties/GlobalDynamicProperties.java

```diff
@@ -157,6 +158,7 @@ public void reload() {
         maxBatchSizeMint = properties.getIntProperty("tokens.nfts.maxBatchSizeMint");
         maxBatchSizeWipe = properties.getIntProperty("tokens.nfts.maxBatchSizeWipe");
         maxNftQueryRange = properties.getLongProperty("tokens.nfts.maxQueryRange");
+        allowTreasuryToOwnNfts = properties.getBooleanProperty("tokens.nfts.useTreasuryWildcards");
```

Comment:
Please merge back `master`, since all the strings are changed to constants in recent PR. And also change this `"tokens.nfts.useTreasuryWildcards"` to a constant.